src/services/modelRegistry.ts

```diff
@@ -0,0 +1,22 @@
+import { initializeRegistry } from '../context/initializeRegistry';
+
+const registry = initializeRegistry();
+
+export const MODEL_REGISTRY = {
+  blockchain: {
+    'gpt-4o': {
+      name: 'gpt-4o',
+      tools: registry.getToolDefinitions(),
+    },
```

Comment:
Let's add a: `description` key to this as well. The mocks have a "short description" in the selector.  I think we should also add/build a `systemPrompt` key and set it up `getSystemPrompt()` for the models that need them